chrome/uninstall.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="uninstall.js"></script>
+    <title></title>
+  </head>
+  <body>
+    <button id="uninstall">Uninstall</button>
```

Comment:
Can you add the extension's name here? "Uninstall" isn't informative enough, especially given that the omnibox will display an opaque key's hash.